discord/member.py

```diff
@@ -159,10 +159,12 @@ class Member(discord.abc.Messageable, _BaseUser):
     premium_since: Optional[:class:`datetime.datetime`]
         A datetime object that specifies the date and time in UTC when the member used their
         Nitro boost on the guild, if available. This could be ``None``.
+    user_flags: list[:class:`UserFlags`]
```

Comment:
This is actually not exposed anywhere. There is no `user_flags` attribute anywhere here.